build/Nuget/Sdkproject.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Would the targets be included in a non-sdk csproj? Or do you mean call the target in the bin dir?